src/HandBrake.php

```diff
@@ -0,0 +1,65 @@
+<?php
+declare(strict_types=1);
+namespace dje\AutoGourcer;
+
+/**
+ * Class HandBrake
+ * @package dje\AutoGourcer
+ */
+class HandBrake
+{
+
+    /**
+     * @var
+     */
+    public $path = './renders';
+
+    /**
+     * @var
+     */
+    public $file = '';
+
+    /**
+     * @var
+     */
+    public $ext = 'mp4';
+
+    /**
+     * @return bool
+     * @throws \Exception
+     */
+    public function transcodeAll(): bool
+    {
+        $dirFiles = glob("$this->path/*.mp4");
+        foreach ($dirFiles as $filePathName) {
+            // remove old file, rename new file to the old's value if transcoding is successful
+            $this->transcode($filePathName);
+        }
+
+        return true;
+    }
```

Comment:
Is it actually necessary for this function to return a boolean here? There isn't a path to return false.

Reply:
Also, in keeping with design pattern, public methods should return $this.